qdrant-landing/content/documentation/concepts/indexing.md

```diff
@@ -228,32 +228,31 @@ The HNSW parameters can also be configured on a collection and named vector
 level by setting [`hnsw_config`](../indexing/#vector-index) to fine-tune search
 performance.
 
-## Sparse vector index
+## Sparse Vector Index
 
-*Available as of v1.7.0*
+**Version Availability:** Available as of v1.7.0
```

Comment:
I think we have the format `*Available as of Vx` all over the docs.
Why changing this one?

Reply:
The existing one is quite hard to read and people have missed it in the past on GH and Discord. I want to make a stab at making it more visible and harder to miss. 

If you think this is the wrong place for this experiment, let's keep it as we have in rest of the docs.

Reply:
This feedback is an interesting data point we should consider for a refactoring :+1: